cli/cmd/cmd.go

```diff
@@ -206,6 +206,8 @@ func (c *ServicedCli) cmdInit(ctx *cli.Context) error {
 		fmt.Fprintf(os.Stderr, "Unable to set logging options: %s\n", err)
 	}
 
+	isvcs.InitAllIsvcs()
```

Comment:
This should move because we don't want/need to initialize ISVCS when running `serviced` as a CLI tool - only when we're starting the actual server process.  This should be moved to `cli/api/daemon.go` - either the `startISVCS()` method or possibly the `run()` method before calling `startISVCS()`

Better yet - can it be called in `isvcs.Init()` instead?